<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Finder</class>
 <widget class="QWidget" name="Finder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QLineEdit" name="findEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Find</string>
     </property>
     <property name="statusTip">
      <string>Find</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Find...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="findPrevious">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Find Previous</string>
       </property>
       <property name="statusTip">
        <string>Find Previous</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/icons/find-previous-pressed.png</normaloff>:/icons/find-previous-pressed.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="findNext">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Find Next</string>
       </property>
       <property name="statusTip">
        <string>Find Next</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/icons/find-next-pressed.png</normaloff>:/icons/find-next-pressed.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="replaceEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Replace With</string>
     </property>
     <property name="statusTip">
      <string>Replace With</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Replace with...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="replacePrevious">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Replace Previous</string>
       </property>
       <property name="statusTip">
        <string>Replace Previous</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/icons/find-previous-pressed.png</normaloff>:/icons/find-previous-pressed.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="replaceNext">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Replace Next</string>
       </property>
       <property name="statusTip">
        <string>Replace Next</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/icons/find-next-pressed.png</normaloff>:/icons/find-next-pressed.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolButton" name="replaceAll">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Replace All</string>
     </property>
     <property name="statusTip">
      <string>Replace All</string>
     </property>
     <property name="text">
      <string>Replace All</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="matchCase">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Match Case</string>
       </property>
       <property name="statusTip">
        <string>Match Case</string>
       </property>
       <property name="text">
        <string>Match Case</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="wholeWord">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Whole Word</string>
       </property>
       <property name="statusTip">
        <string>Whole Word</string>
       </property>
       <property name="text">
        <string>Whole Word</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolButton" name="doneButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
